foundry.toml

```diff
@@ -2,6 +2,6 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-solc_version = "0.8.17"
+solc_version = "0.8.15"
```

Comment:
am on slow connection, foundry kept timing out trying to dl solc, so i jsut temp switched versions to one i had locally